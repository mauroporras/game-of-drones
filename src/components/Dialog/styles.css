.Dialog {
  background-color: var(--color-white);
  border-radius: var(--border-radius-primary);
  box-shadow: 0.2rem 0.2rem 1rem hsla(0, 0%, 10%, 0.4);
  margin-bottom: 2rem;
  padding-bottom: 1rem;
}
