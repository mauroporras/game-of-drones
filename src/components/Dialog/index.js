import React from 'react'

import './styles.css'

const Dialog = props => (
  <div className='Dialog'>
    {props.children}
  </div>
)

export default Dialog
