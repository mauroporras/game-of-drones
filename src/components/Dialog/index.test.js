import React from 'react'
import renderer from 'react-test-renderer'

import Dialog from './index'

test("Component renders correctly", () => {
  const component = renderer.create(
    <Dialog />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
