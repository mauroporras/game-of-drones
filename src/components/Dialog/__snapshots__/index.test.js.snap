exports[`test Component renders correctly 1`] = `
<div
  className="Dialog" />
`;
